astrowidgets/core.py

```diff
@@ -619,6 +619,29 @@ def _validate_marker_name(self, marker_name):
                              '{}'.format(marker_name,
                                          ', '.join(RESERVED_MARKER_SET_NAMES)))
 
+    def add_regions(self, regions):
+        """
+        Creates markers in the image from given regions from
+        the Astropy ``regions`` package.
+
+        .. todo::
+
+            How to tied this into existing marker machinery?
+
+        Parameters
+        ----------
+        regions : obj
+            ``regions`` shapes.
+
+        """
+        from ginga.util import ap_region
+
+        # Regions will be appended if other regions were previously
+        # added in another call.
+        # TODO: How to get or set the canvas name?
+        for reg in regions:
+            ap_region.add_region(self._viewer.canvas, reg)
```

Comment:
Is this too lazy? Should I be using a lower-level Ginga function that is `astropy_region_to_ginga_canvas_object`?